independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json

```diff
@@ -0,0 +1,50 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "bb83f7e1-b028-4115-a5a4-39e02177574c",
```

Comment:
@techienickb Please replace clientID value with "Dummy" value as it is not allowed to expose any security related information.